CMakeLists.txt

```diff
@@ -197,13 +197,23 @@ else( WIN32 ) # Apple AND Linux
         message( STATUS "Configuring BitShares on OS X" )
         set( CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -stdlib=libc++ -Wall" )
     else( APPLE )
-        # Linux Specific Options Here
-        message( STATUS "Configuring BitShares on Linux" )
+        if ( "${CMAKE_SYSTEM_NAME}" STREQUAL "OpenBSD" )
+            # OpenBSD Specific Options
+            message( STATUS "Configuring BitShares on OpenBSD" )
+            if ( "${BOOST_ROOT}" STREQUAL "" )
+                # linking with system boost on OBSD is dynamic
+                set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_TEST_DYN_LINK" )
```

Comment:
IMO this is a bad hack. Concluding that boost libs are static because it's OpenBSD and BOOST_ROOT is not set is probably wrong. Fortunately, this can be removed if you use `BOOST_TEST_MODULE` unconditionally, as suggested.

Reply:
Can't disagree. Fortunately, it is now gone.